### PR TITLE
Update README

### LINKED ISSUE #52: update incident management section in README
Update incident management section of readme: https://github.com/operate-first/SRE/blob/master/README.md#incident-management

### LINKED ISSUE #933: update incident management section in README
Update incident management section of readme: https://github.com/operate-first/SRE/blob/master/README.md#incident-management

### PR DESCRIPTION
**Pull request by hemajv**
this closes operate-first/support#933

### CODE REVIEW

**Review by anishasthana — APPROVED** [review]
/lgtm

File: README.md [hunk 1]
```diff
@@ -15,21 +15,27 @@ The components we currently have available are:
...
 
-## Incident Management
+The [`incident-management`](https://github.com/operate-first/SRE/tree/master/incident-management) folder contains more information on how to setup GitHub alertmanager receiver and configure Prometheus alerts.
+
+## Runbooks
+
+For all the services being monitored, we define runbooks to provide information on how to administer, debug and effectively troubleshoot common problems of a service.
 
-TO DO
+The runbook for each of the individual services are defined in the [`runbooks`](https://github.com/operate-first/SRE/tree/master/runbooks)
```
Comment (HumairAK) [inline README.md]: Can we change the wording here to sound more like we're aiming, or  working towards adding runbooks for monitored services, and we'll add more as time goes on. Atm it sounds like we have already done this for all monitored service (when we haven't).

File: README.md [hunk 2]
```diff
@@ -15,21 +15,27 @@ The components we currently have available are:
...
+For each of the Operate First managed services, we define Service Level Indicators (SLI) and Service Level Objectives (SLO) to help us monitor the availability of the service and improve its reliability.
 
 The `sli-slo` folder [here](https://github.com/operate-first/SRE/tree/master/sli-slo) lists out the SLI/SLO for each of the applications in separate markdown files. For eg, you can find the SLI/SLO defined for the `JupyterHub` application [here](https://github.com/operate-first/SRE/blob/master/sli-slo/jupyterhub.md).
 
 ## Operational Tools
 
-TO DO
+* [Prometheus](http://prometheus-portal-opf-monitoring.apps.cnv.massopen.cloud/graph) - Monitoring tool capturing time series metrics for available Operate First services
+* [Grafana](https://grafana-route-opf-monitoring.apps.cnv.massopen.cloud/) - Visualization tool for creating monitoring dashboards
+* [GitHub Receiver](https://github.com/m-lab/alertmanager-github-receiver) - Incident reporting tool for handling outages/incidents
```
Comment (4n4nd) [inline README.md:27]: Can we add something about alertmanager? Because we can support other types of receivers too

File: README.md (hunk 1 above)
Comment (hemajv) [reply README.md]: updated :+1:

File: README.md (hunk 2 above)
Comment (hemajv) [reply README.md:27]: good idea! added :+1:

**Review by HumairAK — APPROVED** [review]

**Comment by HumairAK** [comment]
Bah I didn't notice there was already an approve, woops.

**Pull request merged**